dashboard-ui/src/components/TextBasedScenarios/util.js

```diff
@@ -0,0 +1,183 @@
+const axios = require('axios');
+const { stUrban, stDesert, stJungle, stSubmarine } = require('./problemProbes')
+// add proper choice id's to questions
+const mapAnswers = (scenario, scenarioMappings) => {
+    if (!scenario.participantID) { return; }
+    const scenarioConfig = scenarioMappings[scenario.title.replace(' Scenario', '')];
+    Object.entries(scenario).forEach(field => {
+        if (!field[1]?.questions) { return; }
+        Object.entries(field[1].questions).forEach(question => {
+            if (!question[1].probe) { return; }
+            const page = scenarioConfig.pages.find((page) => page.name === field[0]);
+            const pageQuestion = page.elements.find((element) => element.name === question[0]);
+            const indexOfAnswer = pageQuestion.choices.indexOf(question[1].response);
+            // should never happen for real data but don't want older test data throwing errors
+            if (indexOfAnswer < 0) { return; }
+            let choice;
+            if (scenario.title.includes("Adept")) {
+                if (indexOfAnswer >= 0) {
+                    choice = question[1].probe + '.';
+                    choice += String.fromCharCode(65 + indexOfAnswer);
+                } else {
+                    console.error("Error mapping user selection to choice ID");
+                }
+            } else {
+                choice = "choice-" + indexOfAnswer;
+            }
+            question[1].choice = choice;
+        })
+    })
+};
+
+// add proper probe id's to questions
+const addProbeIDs = (scenario, scenarioMappings) => {
+    if (!scenario.participantID) { return; }
+    const scenarioConfig = scenarioMappings[scenario.title.replace(' Scenario', '')];
+    Object.entries(scenario).forEach(field => {
+        if (!field[1]?.questions) { return; }
+        Object.entries(field[1].questions).forEach(question => {
+            if (question[1].response && !question[0].includes("Follow Up")) {
+                const page = scenarioConfig.pages.find((page) => page.name === field[0]);
+                const pageQuestion = page.elements.find((element) => element.name === question[0]);
+                question[1].probe = pageQuestion.probe;
+            }
+        })
+    })
+}
+
+const getAdeptAlignment = async (scenarioResults, scenarioId) => {
+    /*local host 8080 is where I had adept running, if you had to change your port must change script to match */
+    // build fetch for starting new session
+    const url = 'http://127.0.0.1:8080/api/v1/new_session';
```

Comment:
So this script isn't going to work on production because the TA1 servers are on different machines.  I think we need to get prod urls to test this, and we might need to be on the VPN to do it.